spec/spec_helper.rb

```diff
@@ -8,6 +8,33 @@
 
 require 'voxpupuli/test/spec_helper'
 
+# Rough conversion of grepping in the puppet source:
```

Comment:
Won't modulesync override this?

Btw, I've been considering adding this to voxpupuli-test given I've repeated this a few times. I'll see about refreshing that code and sending a PR.

Reply:
Yes. That would be my next question, but you were to quick. I would love to see this at voxpupuli-test, because it is very generic and extremely useful.